packages/core/src/App/Containers/Modals/cooldown-warning-modal.jsx

```diff
@@ -25,9 +25,16 @@ const CooldownWarningModal = ({
                 <Icon icon='IcRedWarning' size={63} />
                 <Text as='p' size='xs' align='center' className='risk-acceptance__text'>
                     <Localize
-                        i18n_default_text='You can create your account on {{real_account_unblock_date}}. <0/>Please click ‘OK’ to continue.'
+                        i18n_default_text="Sorry, you're unable to create an account at this time. As you declined our previous risk warnings, we need you to wait for 24 hours after your first account creation attempt before you can proceed.<0/>"
+                        components={<div key={0} style={{ marginBottom: '1rem' }} />}
```

Comment:
No in line css, please handle it using the classname or add multiple Text component.

Reply:
@matin-deriv Already made the changes in the latest commit 👍🏼